### PR TITLE
Add pan functionality to home histograms

### PR DESCRIPTION
**Pull request by argaen**
Adding ability to pan using zoom chartjs plugin.

It's not the smoothest but best than nothing. In future version we probably want to add a secondary chart similar to stocks chart so you can scroll/select dates

### CODE REVIEW

File: public/books/demo.sqlite.gz
Comment (argaen) [inline public/books/demo.sqlite.gz:1]: Re-uploading as it didn't update properly...

File: src/app/dashboard/accounts/page.tsx [hunk 1]
```diff
@@ -19,6 +18,9 @@ import DateRangeInput from '@/components/DateRangeInput';
 import Onboarding from '@/components/onboarding/Onboarding';
 import * as API from '@/hooks/api';
 
+const NetWorthHistogram = dynamic(() => import('@/components/pages/accounts/NetWorthHistogram'), { ssr: false });
+const IncomeExpenseHistogram = dynamic(() => import('@/components/pages/accounts/IncomeExpenseHistogram'), { ssr: false });
```
Comment (argaen) [inline src/app/dashboard/accounts/page.tsx:22]: need to import like this because zoom plugin needs to be frontend rendered

**Comment by ruru-m07** [comment]
nice work!

**Comment by ruru-m07** [comment]
Done!

**Pull request merged**